### DISCUSSION TITLE
Animation when closing Dialog

### DISCUSSION THREAD

**Discussion by dsx20** [fgilde/MudBlazor.Extensions#79] [Ideas]
Is there any way to achieve an animation (e.g. fade out) when closing a dialog?

**Comment by fgilde** [comment]
No, not yet.  Maybe I can add that feature, currently MudBlazor removes the reference immediatly but I will try to implement that

**Comment by fgilde** [comment]
So I added this Feature it will be released in 1.7.88 for testing its avaliable now in this preview packege https://www.nuget.org/packages/MudBlazor.Extensions/1.7.88-prev-2405261413-main

See using sample https://mudex.azurewebsites.net/dialogs
